src/app.rs

```diff
@@ -1436,7 +1436,7 @@ This flag can be used with the -o/--only-matching flag.
 fn flag_search_zip(args: &mut Vec<RGArg>) {
     const SHORT: &str = "Search in compressed files.";
     const LONG: &str = long!("\
-Search in compressed files. Currently gz, bz2, xz, and lzma files are
+Search in compressed files. Currently gz, bz2, xz, lzma, and lz4 files are
```

Comment:
Can we sort this list alphabetically, just like it is in `FAQ.md`?

(Sorry for being so OCD!)

Reply:
I kept the original ordering because it seemed to be by popularity